### DISCUSSION TITLE
Indicate if a recipe can be frozen or kept in storage (and for how long)

### DISCUSSION THREAD

**Discussion by gabt92** [mealie-recipes/mealie#821] [Feature Request]
**Before submitting this feature request I have**
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
We can only indicate in the description or in the notes for how long a recipe can be kept frozen or in storage (if it's the case). 

**Suggest A Solution**
Add fields on the recipe entity to be able to indicate of the recipe can be frozen or kept in storage and, if it's the case, for how long.
```
{
  ...
  "totalTime": "65 minutes",
  "prepTime": "45 minutes",
  "cookTime": null,
  "maxFrozenTime": "3 weeks",
  "maxStorageTime": "2 days",
  ...
}
```
Like to totalTime, prepTime and cookTime, if the fields are filled, the data could be shown in the image as follow:
![image](https://user-images.githubusercontent.com/38195308/143078865-09658f5f-9ff7-4636-8075-7e7e21505082.png)

Keeping the fields empty would be considering that the recipe can't be frozen or kept in storage.

On the screens with many recipes, an icon could be displayed to easily indicate which one can be frozen or not. Small mockup (the icon could be more visible than that and I'm not sure about a logo for storage...): 
![image](https://user-images.githubusercontent.com/38195308/143080533-61147174-61b4-4328-baae-93d5e1f74297.png)

In the advanced search, it could be possible to filter by recipes than can be frozen and/or stored (even possibly indicate the time needed, but since the time entries are not controlled, this can become tricky):
![image](https://user-images.githubusercontent.com/38195308/143081114-c282e14a-85dc-4cf0-b48a-a4a687597f58.png)

**Additional Information**
- [X] If the feature is accepted I'm willing to submit a PR to provide this feature
> I can try to do my best, I'll probably need help but I'm willing to learn.

**Comment by hay-kot** [comment]
I'm not sure I'm in favor of an additionally attribute for this, especially with all the additional UI elements and Data that would need to be managed. 

My feeling is that Tags would accomplish 90% of what's needed here already, the additional Freeze/Storage time is a pretty niche feature-set that wouldn't be widely used.

**Comment by michael-genson** [comment]
Closing this per hay-kot above; this can be achieved with existing organizers
